source/index.rst

```diff
@@ -23,7 +23,7 @@ Select one of the headings to get started!
 .. toctree::
    :maxdepth: 1
 
-   beginner 
+   fr-beginner
```

Comment:
please don't commit this change - I'm planning a way that we can build the various languages stand alone - but the default langauge should be english